docs/getting-started/ory-network-oauth2.mdx

```diff
@@ -1,10 +1,10 @@
 ---
 id: ory-network-oauth2
 title: Perform OAuth2 Authorization Code Grant and Client Credentials Grant
-sidebar_label: Try it
+sidebar_label: OAuth2
 ---
 
-# Try common OAuth2 Grants
```

Comment:
With the new content, this page probably needs to be refreshed or changed? Maybe move it under get started and adjust it to have "Test client credentials grant" and "Test code grant" or something like that where the user, after setting up their project and the client, can check what the flow looks like without code?